pybossa/view/projects.py

```diff
@@ -3121,7 +3121,7 @@ def sync_project(short_name):
         current_app.logger.info(
             'exception type is {}'
             .format(str(exception_type)))
-        msg = gettext('An unexpected error occurred while trying to sync your project')
+        msg = gettext('Error: Ensure PROD API key is used, your PROD account is Subadmin & enabled, & target project is enabled for project syncing')
```

Comment:
This looks good, although I am thinking if the error message could be made more clear by changing `&` to `and`.

Reply:
sure, i can do this now. 
I was using the language provided by Tim. If you think any other changes are necessary, let me know!